Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.Animator.cs

```diff
@@ -0,0 +1,38 @@
+using ECM2.Characters;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public partial class FirstPersonPlayer : FirstPersonCharacter
```

Comment:
```suggestion
    public partial class FirstPersonPlayer
```
You can remove the `:FirstPersonCharacter` part. since the original class already derives from it